rust/src/utils.rs

```diff
@@ -304,6 +309,87 @@ pub(crate) fn translate_path_for_ostree_impl(path: &str) -> Option<String> {
     None
 }
 
+#[context("Deduplicate tmpfiles entries")]
+pub fn deduplicate_tmpfiles_entries(tmprootfs_dfd: i32) -> CxxResult<()> {
```

Comment:
There's sufficient related code here that it may call for a new `mod tmpfiles` or so?